Generic.c

```diff
@@ -0,0 +1,102 @@
+/*
+htop - Generic.c
+(C) 2021 htop dev team
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+#include "config.h"  // IWYU pragma: keep
+
+#include "Generic.h"
+
+#include <stdio.h>
+#ifdef HAVE_SYS_UTSNAME_H
+#include <sys/utsname.h>
+#endif
```

Comment:
Maybe put this include last, cf. styleguide. Also two blank lines after includes.